src/OpenLoco/Windows/TitleVersion.cpp

```diff
@@ -30,7 +30,6 @@ namespace OpenLoco::Ui::Windows::TitleVersion
             &_events);
         window->widgets = widgets;
 
-        _events.prepare_draw = (void (*)(Ui::Window*))0x0042A035;
```

Comment:
Oh wow. We set a vanilla routine to our own custom window? Amazing

Reply:
its just a ret

Reply:
(`prepare_draw` wasn't implemented yet this early on)